flowstate_ros_bridge/src/executive.cpp

```diff
@@ -360,8 +363,11 @@ auto Executive::ProcessHandle::make(
                   client_context.get(), std::move(get_request),
                   &handle->current_operation_);
               if (!status.ok()) {
-                LOG(INFO) << "[process thread] Error while getting operation "
-                          << handle->current_operation_.name();
+                std::stringstream ss;
+                ss << "[process thread] Error while getting operation "
+                   << handle->current_operation_.name();
+                LOG(INFO) << ss.str();
+                handle->feedback_cb_(absl::UnavailableError(ss.str()));
               }
               if (handle->current_operation_.done()) {
```

Comment:
Before this block, we can publish a feedback `handle->feedback_cb_(handle->current_operation_.done(), status.ok());`